doc/source/tutorial/optimize.rst

```diff
@@ -368,106 +368,129 @@ and a constrained optimization as:
     [ 1.00000009  1.        ]
 
 
-Least-square fitting (:func:`leastsq`)
---------------------------------------
+Least-squares minimization (:func:`least_squares`)
+--------------------------------------------------
 
-All of the previously-explained minimization procedures can be used to
-solve a least-squares problem provided the appropriate objective
-function is constructed. For example, suppose it is desired to fit a
-set of data :math:`\left\{\mathbf{x}_{i}, \mathbf{y}_{i}\right\}`
-to a known model,
-:math:`\mathbf{y}=\mathbf{f}\left(\mathbf{x},\mathbf{p}\right)`
-where :math:`\mathbf{p}` is a vector of parameters for the model that
-need to be found. A common method for determining which parameter
-vector gives the best fit to the data is to minimize the sum of squares
-of the residuals. The residual is usually defined for each observed
-data-point as
+SciPy is capable of solving robustified bound constrained nonlinear
+least-squares problems:
 
 .. math::
-   :nowrap:
-
-    e_{i}\left(\mathbf{p},\mathbf{y}_{i},\mathbf{x}_{i}\right)=\left\Vert \mathbf{y}_{i}-\mathbf{f}\left(\mathbf{x}_{i},\mathbf{p}\right)\right\Vert
-
-An objective function to pass to any of the previous minimization
-algorithms to obtain a least-squares fit is.
+  \begin{align}
+  &\min_\mathbf{x} \frac{1}{2} \sum_{i = 1}^m \rho\left(f_i(\mathrm{x})^2\right) \\
+  &\text{s. t. } \mathbf{lb} \leq \mathbf{x} \leq \mathbf{ub}
```

Comment:
I thought of "subject to". I guess it is not really a valid acronym, not sure where I saw it used.